### PR TITLE
build: Tag CSP resources according to policy

### PR DESCRIPTION
**Pull request by kruskall**
Reuse apm-server tags module to share labels for the apm-server team. Tag resource accordingly.

Related to https://github.com/elastic/apm-server/issues/9699

### CODE REVIEW

File: testing/benchmarking/main.tf [hunk 1]
```diff
@@ -33,6 +33,10 @@ provider "aws" {
   region = var.aws_region
```
Comment (marclop) [inline testing/benchmarking/main.tf:33]: All the AWS resources will be tagged if you specify a `default_tags` field: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags

Comment (kruskall) [reply testing/benchmarking/main.tf:33]: Do you mean instead of passing the tags to the tf modules ? I don't think we can assume they are using aws

**Comment by kruskall** [comment]
@marclop Thanks for the help and discussion with this!

I've pushed some changes

File: tf/variables.tf [hunk 2]
```diff
@@ -4,6 +4,11 @@ variable "aws_region" {
   default     = "eu-central-1"
 }
 
+variable "user_name" {
+  description = "Required username to use for prefixes"
+  type        = string
+}
+
```
Comment (marclop) [inline tf/variables.tf:11]: Should this variable be also removed? It doesn't seem to be used.

**Review by marclop — APPROVED** [review]
Looks good!

File: tf/variables.tf (hunk 2 above)
Comment (kruskall) [reply tf/variables.tf:11]: I think it's used here: https://github.com/elastic/apm-aws-lambda/pull/355/files#diff-a2fd895ac25407846fd26b6d4fc9a4482f8cab6767431f76535be5ab0abe15eeR7 to set the project field.

Comment (marclop) [reply tf/variables.tf:11]: never mind then!

**Pull request merged**